src/Illuminate/Contracts/Cache/Repository.php

```diff
@@ -3,8 +3,9 @@
 namespace Illuminate\Contracts\Cache;
 
 use Closure;
+use Psr\SimpleCache\CacheInterface;
 
-interface Repository
+interface Repository extends CacheInterface
```

Comment:
Yeah true, it' weird. I had a look at `\Illuminate\Contracts\Container\Container` too which extends `\Psr\Container\ContainerInterface`